Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -81,6 +81,25 @@ public protocol MessagesLayoutDelegate: AnyObject {
     ///   The default value returned by this method is zero.
     func messageBottomLabelHeight(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> CGFloat
 
+    /// Specifies the size for the `MessageContentCell`'s accessory view.
+    ///
+    /// - Parameters:
+    ///   - message: The `MessageType` that will be displayed for this cell.
+    ///   - messagesCollectionView: The `MessagesCollectionView` in which this cell will be displayed.
+    ///
+    /// - Note:
+    ///   The default value returned by this method is zero.
+    func accessoryViewSize(for message: MessageType, in messagesCollectionView: MessagesCollectionView) -> CGSize
+
+    /// Specifies the padding for the `MessageContentCell`'s accessory view.
+    ///
+    /// - Parameters:
+    ///   - message: The `MessageType` that will be displayed for this cell.
+    ///   - messagesCollectionView: The `MessagesCollectionView` in which this cell will be displayed.
+    ///
+    /// - Note:
+    ///   The default value returned by this method is zero.
+    func accessoryViewPadding(for message: MessageType, in messagesCollectionView: MessagesCollectionView) -> UIEdgeInsets
```

Comment:
Padding would definitely need to be defined through the cell calculator objects and not via delegate